rising/transforms/affine.py

```diff
@@ -572,16 +574,105 @@ def assemble_matrix(self, **data) -> torch.Tensor:
 
         """
         whole_trafo = None
-
         for trafo in self.transforms:
             matrix = matrix_to_homogeneous(trafo.assemble_matrix(**data))
-
             if whole_trafo is None:
                 whole_trafo = matrix
             else:
                 whole_trafo = torch.bmm(whole_trafo, matrix)
-
         return matrix_to_cartesian(whole_trafo)
 
-# TODO: Add transforms around image center
-# TODO: Add Resize Transform
+
+class Resize(Scale):
```

Comment:
Probably I won't insert this trafo in his PR, but in #29 , where we will sort out everything else and keep this as a refactoring only-PR